### ISSUE TITLE
[Feature request] Add "empty" parameter for Pandas DataFrame converter

### ISSUE THREAD

**Issue by mullimanko** [xlwings/xlwings#2360]
I would like to suggest a parameter to the Pandas DataFrame and Series converter: The parameter `emtpy` to convert `pd.NA` and `np.nan` to the argument specified. For example:

```
import xlwings as xw
import pandas as pd
import numpy as np

df = pd.DataFrame([[1, 2, 3],
                   [np.nan, pd.NA, 0],
                   ])
# df:
#      0     1  2
# 0  1.0     2  3
# 1  NaN  <NA>  0

wb = xw.Book()
ws = wb.sheets[0]
ws["A1"].options(empty="NA").value = df

# This should be inserted in Excel as:
#      0     1  2
# 0  1.0     2  3
# 1  NA     NA  0
```
Particularly when the index of a DataFrame or Series contains `pd.NA` or `np.nan`, using such an parameter would reduce difficulties in interpreting a table, because NA (or something else) is expicitly specified. Current behaviour is that `pd.NA` and `np.nan` are converted to empty cells.

**Comment by fzumstein** [comment]
Sounds good, but not sure about calling it `empty` and I would make this generic, not just for DataFrames so it can work with `None` also. Possibly something like `null`, `na` or `nan` might make more sense?

**Comment by mullimanko** [comment]
Yes great, if I may give some suggestions, borrowed from pandas: [na](https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html#experimental-na-scalar-to-denote-missing-values) or [fillna](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.fillna.html).

**Comment by fzumstein** [comment]
agreed, probably `myseet["A1"].options(na="N/A").value = [1, None, 3]` would be my favorite.
